index.d.ts

```diff
@@ -0,0 +1,62 @@
+import {IOptions as GlobOptions} from 'glob';
+
+interface Options extends Readonly<GlobOptions> {
+	/**
+	 * Allow deleting the current working directory and outside.
+	 *
+	 * @default false
+	 */
+	readonly force?: boolean;
+
+	/**
+	 * See what would be deleted.
+	 *
+	 * @default false
+	 *
+	 * @example
+	 *
+	 * const del = require('del');
```

Comment:
Use `import` syntax. This is supposed to be a TS usage example.

```suggestion
	 * import del from 'del';
```